scripts/codegen/README.md

```diff
@@ -0,0 +1,2 @@
+generate typescript types for the grep team. Note that this does not create endpoints, and may re-defined already defined common types found in the sdk or core.
```

Comment:
Can we write a proper Markdown formatted README?

Reply:
Maybe we could also describe a bit more about how to run the codegen and, if we need need something else in order to run it, describe it too.

Reply:
some of the output errors are related to cognite tooling, the oas merger, which are expected for playground.